test/common/upstream/load_balancer_simulation_test.cc

```diff
@@ -81,26 +113,88 @@ TEST(DISABLED_LeastRequestLoadBalancerWeightTest, Weight) {
       priority_set, nullptr, lb_stats, runtime, random, common_config, least_request_lb_config,
       *time_source};
 
-  absl::node_hash_map<HostConstSharedPtr, uint64_t> host_hits;
-  const uint64_t total_requests = 100;
-  for (uint64_t i = 0; i < total_requests; i++) {
+  for (uint64_t i = 0; i < n; i++) {
```

Comment:
I find it strange, as `n` is no longer the number of `total_requests`.
Can you add some comments clarifying what is the intention here?